guide/webpack.config.js

```diff
@@ -32,6 +33,15 @@ module.exports = validate({
                 include: RegExp(__dirname + "/src"),
                 loaders:["react-hot", "babel-loader"]
             },
+	    {
+		test: /\.txt$/,
+		loader: 'raw-loader',
+		include: path.resolve(__dirname, 'src/examples/raw-code'),
+	    },
+	    {
+		test: /\.md$/,
+		loader: 'raw-loader',
+	    },
```

Comment:
The points brought up here by @lenards have been addressed in [this PR](https://github.com/cyverse/cyverse-ui/pull/43)